src/main/java/uk/gov/ch/pocappointmentapi/Appointment.java

```diff
@@ -0,0 +1,226 @@
+		this.appointmentId = appointmentId;
+	}
+
+	public String getOfficerId() {
+		return officerId;
+	}
+
+	public void setOfficerId(String officerId) {
+		this.officerId = officerId;
+	}
+
+	public String getCompanyNumber() {
+		return companyNumber;
+	}
+
+	public void setCompanyNumber(String companyNumber) {
+		this.companyNumber = companyNumber;
+	}
+
+	public String getCompanyName() {
+		return companyName;
+	}
+
+	public void setCompanyName(String companyName) {
+		this.companyName = companyName;
+	}
+
+	public String getCompanyStatus() {
+		return companyStatus;
+	}
+
+	public void setCompanyStatus(String companyStatus) {
+		this.companyStatus = companyStatus;
+	}
+
+	public Data getData() {
+		return data;
+	}
+
+	public void setData(Data data) {
+		this.data = data;
+	}
+
+	@Override
+	public String toString() {
+		return "Appointment [appointmentId=" + appointmentId + ", officerId=" + officerId + ", companyNumber="
+				+ companyNumber + ", companyName=" + companyName + ", forename=" + data.forename + ", surname=" + data.surname
+				+ "]";
+	}
+	
+	class Data{
+		@Field(value = "title")
+	    private String title;
+		
+	    @Field(value = "forename")
+	    private String forename;
+
+	    @Field(value = "surname")
+	    private String surname;
+	    
+	    @Field(value="date_of_birth")
+	    private String dateOfBirth;
+	    
+	    @Field(value="officer_role")
+	    private String officerRole;
+	    
+	    @Field(value="service_address")
+	    private ServiceAddress serviceAddress;
+	    
+	    @Field(value="appointed_on")
+	    private String appointedOn;
+	    
+	    @Field(value="updated_at")	    
+	    private String updatedAt;
+	    
+	    
+		public String getTitle() {
+			return title;
+		}
+
+		public void setTitle(String title) {
+			this.title = title;
+		}
+
+		public String getForename() {
+			return forename;
+		}
+
+		public void setForename(String forename) {
+			this.forename = forename;
+		}
+		
+		public String getSurname() {
+			return surname;
+		}
+
+		public void setSurname(String surname) {
+			this.surname = surname;
+		}
+
+		public String getDateOfBirth() {
+			return dateOfBirth;
+		}
+
+		public void setDateOfBirth(String dateOfBirth) {
+			this.dateOfBirth = dateOfBirth;
+		}
+
+		public String getOfficerRole() {
+			return officerRole;
+		}
+
+		public void setOfficerRole(String officerRole) {
+			this.officerRole = officerRole;
+		}
+
+		public ServiceAddress getServiceAddress() {
+			return serviceAddress;
+		}
+
+		public void setServiceAddress(ServiceAddress serviceAddress) {
+			this.serviceAddress = serviceAddress;
+		}
+		
+		public String getAppointedOn() {
+			return appointedOn;
+		}
+
+		public void setAppointedOn(String appointedOn) {
+			this.appointedOn = appointedOn;
+		}
+		
+		public String getUpdatedAt() {
+			return updatedAt;
+		}
+
+		public void setUpdatedAt(String updatedAt) {
+			this.updatedAt = updatedAt;
+		}
+
+		class ServiceAddress{
```

Comment:
Please move this inner instance-inner instance class out into a separate file.

Reply:
Done